src/tokens/selectors.ts

```diff
@@ -467,6 +467,15 @@ export const sortedTokensWithBalanceOrShowZeroBalanceSelector = createSelector(
     })
 )
 
+export const sortedCkesandCusdSelector = createSelector(
```

Comment:
nit: this function doesn't have anything to do with cUSD specifically. Maybe `cKESFirstTokensListSelector` ?